src/sentry/static/sentry/app/views/projectDashboard/eventList.jsx

```diff
@@ -12,8 +13,9 @@ const EventList = createReactClass({
   displayName: 'EventList',
 
   propTypes: {
-    title: PropTypes.string.isRequired,
-    endpoint: PropTypes.string.isRequired,
+    type: PropTypes.oneOf(['new', 'priority']).isRequired,
+    environment: PropTypes.object,
```

Comment:
future: Would be helpful to have this described as a `shape` in our `proptypes` module.